app/helpers/form_helper.rb

```diff
@@ -104,27 +104,28 @@ def radio_button_f(f, attr, options = {})
     end
   end
 
-  def select_f(f, attr, array, id, method, select_options = {}, html_options = {})
+  def select_include_blank(array, include_blank, html_options, select_options)
     array = array.to_a.dup
-    disable_button = select_options.delete(:disable_button)
-    include_blank = select_options.delete(:include_blank)
-    disable_button_enabled = select_options.delete(:disable_button_enabled)
-    user_set = !!select_options.delete(:user_set)
-
     if include_blank
       blank_value = include_blank.is_a?(TrueClass) ? nil : include_blank
-      blank_option = OpenStruct.new({id => '', method => blank_value })
-      # if the method is to_s, OpenStruct will respond with its own version.
-      # in this case, I need to undefine its own alias to to_s, and use the attribute
-      # that was defined in the struct.
-      blank_option.instance_eval('undef to_s', __FILE__, __LINE__) if method.to_s == 'to_s' || id.to_s == 'to_s'
+      blank_option = OpenStruct.new({:id => '', :method => blank_value })
+      blank_option.instance_eval('undef to_s', __FILE__, __LINE__)
```

Comment:
Also there were more ruby complains for `method.to_s == 'to_s' || id.to_s == 'to_s'` so I left it and it seems to still work if blank_value is bool or string so I dont know why it was there?